experimental/singlenumberformatter/src/provider.rs

```diff
@@ -51,6 +51,9 @@ pub struct CurrencyEssentialsV1<'data> {
     /// Contains all the place holders.
     #[cfg_attr(feature = "serde", serde(borrow))]
     pub place_holders: VarZeroVec<'data, str>,
+
+    /// The default pattern.
+    pub default_pattern: DefaultPattern,
```

Comment:
Suggestion (optional): I was thinking this field would be an instance of `CurrencyPatterns` instead of an enum. Then we can support arbitrary defaults, not just the ones that you have in the enum. However, I think these two cases are the only two that are likely to be needed.

Reply:
good idea. I will do it in the followup PR.

Reply:
https://github.com/unicode-org/icu4x/pull/4386